lib/Controller/RoomController.php

```diff
@@ -1482,6 +1483,30 @@ protected function changeParticipantType(int $attendeeId,  bool $promote): DataR
 		return new DataResponse();
 	}
 
+	/**
+	 * @PublicPage
```

Comment:
```suggestion
	 * @NoAdminRequired
```

you dont allow non-logged in users in the next line anyway.